ui/main.qml

```diff
@@ -49,16 +44,12 @@ StatusWindow {
 
     objectName: "mainWindow"
     color: Theme.palette.background
-    title: {
-        // Set application settings
-        Qt.application.name = "Status Desktop"
-        Qt.application.displayName = qsTr("Status Desktop")
-        Qt.application.organization = "Status"
-        Qt.application.domain = "status.im"
-        Qt.application.version = aboutModule.getCurrentVersion()
-        return Qt.application.displayName
-    }
-    visible: true
```

Comment:
Why this change?

Reply:
removed visible properties as it collides with visibility and leads to undefined behaviors and errors are reported in the logs